CONTRIBUTING.md

```diff
@@ -90,7 +91,7 @@ Chrome v59 or higher (v60 if you're on Windows).
 
 If you want to run a single test file, run `npm run mocha ${path/to/spec-file}`,
 substituting the actual path to the spec. Only javascript files can be executed by the `mocha` runner,
-and only those js files in the `spec` directory ending with a `.spec.js`.
+and only those js files in the `src/patterns/` directory ending with a `.spec.js`.
```

Comment:
Thank you for updating the docs!

Should this be `src/patterns/test/`?

Reply:
This could probably be confusing because of the way `patternlab-config.json` is set up right now.

```JSON
  "styleGuideExcludes": [
    "compare",
    "test"
  ],
```

maybe the way to resolve this is to update that config to say,

```JSON
  "styleGuideExcludes": [
    "compare",
    "test-patterns"
  ],
```

then our contrib file would state
```
and only those js files in the `src/patterns/*` directory ending with a `.spec.js`.
```

Reply:
I agree, it is confusing to call it test. Let's rename it to `test-patterns` and we can merge.